.gitignore

```diff
@@ -68,7 +70,8 @@ target/
 
 # misc
 tests/pub
+tests/setup.cfg
```

Comment:
So I needed a way for the tests to get the _main_ `setup.cfg` file for the "kitchen sink" config, so I added this https://github.com/mozilla-services/kinto-dist/pull/1953/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R68. Because of that, it'll write the file to the `tests` directory as well and I didn't want two of the same file, so I added to `.gitignore`.

If there's a better way to go about doing this so the tests get that config file, I'm all ears!